src/model/trust.model.ts

```diff
@@ -33,6 +33,7 @@ export interface Trust {
   ceased_date_day?: string;
   ceased_date_month?: string;
   ceased_date_year?: string;
+  trust_still_involved_in_overseas_entity: string; // May not be relevant in some cases so yesNoResponse cannot be used
```

Comment:
should this be optional too?

Reply:
Here we go again... Yes, probably. Will look into changing this (and any other occurrences).

Reply:
Changed this now.